src/web_info.rs

```diff
@@ -0,0 +1,43 @@
+use crate::error::AybError;
```

Comment:
A few thoughts on naming and organization
* We should consider this living in `src/http/`: it might not look like it now, but in the future, ayb might support other protocols (e.g., psql)
* `WebInfo` -> `WebFrontendDetails` or `WebFrontendConfig`? Trying to be more specific about what web information it is.

Reply:
`WebFrontendDetails` seems more accurate, I'll move the module and update its name

Reply:
Thank you! :)